packages/site-middleware/src/withSearchIndex.ts

```diff
@@ -63,47 +30,53 @@ export const withSearchIndex: MosaicMiddleware<SearchIndexSlice> = async (
   const matches = resolvedUrl.match(/(.*)[!/]/);
   const urlPath = matches?.length ? matches[1] : '';
 
-  const mosaicUrl = isSnapshotS3 || isSnapshotFile ? '' : res.getHeader('X-Mosaic-Content-Url');
-  const searchDataUrl = `${mosaicUrl}/search-data.json`;
-
   try {
-    let searchIndexData;
+    let searchIndex;
     if (isSnapshotFile) {
-      const env = getEnv(urlPath);
-      const { MOSAIC_SNAPSHOT_DIR: mosaicSnapshotDir } = env.data;
-      const filePath = path.join(process.cwd(), mosaicSnapshotDir, 'search-data.json');
-      const rawSearchIndexData = await loadLocalFile(filePath);
-      searchIndexData = JSON.parse(rawSearchIndexData);
+      const { snapshotDir } = getSnapshotFileConfig(urlPath);
+      const filePath = path.join(process.cwd(), snapshotDir, urlPath, 'search-data.json');
```

Comment:
Temp fix to revert namespace support

```suggestion
      const filePath = path.join(process.cwd(), snapshotDir, 'search-data.json');
```